src/commands/Minion/mix.ts

```diff
@@ -0,0 +1,167 @@
+import { CommandStore, KlasaMessage } from 'klasa';
+
+import { BotCommand } from '../../lib/BotCommand';
+import { Activity, Tasks, Time } from '../../lib/constants';
+import { minionNotBusy, requiresMinion } from '../../lib/minions/decorators';
+import { UserSettings } from '../../lib/settings/types/UserSettings';
+import Herblore from '../../lib/skilling/skills/herblore/herblore';
+import { SkillsEnum } from '../../lib/skilling/types';
+import { HerbloreActivityTaskOptions } from '../../lib/types/minions';
+import {
+	bankHasItem,
+	formatDuration,
+	itemNameFromID,
+	removeItemFromBank,
+	stringMatches
+} from '../../lib/util';
+import addSubTaskToActivityTask from '../../lib/util/addSubTaskToActivityTask';
+
+export default class extends BotCommand {
+	public constructor(store: CommandStore, file: string[], directory: string) {
+		super(store, file, directory, {
+			aliases: ['crush', 'clean'],
+			altProtection: true,
+			oneAtTime: true,
+			cooldown: 1,
+			usage: '[quantity:int{1}|name:...string] [name:...string]',
+			usageDelim: ' '
+		});
+	}
+
+	@minionNotBusy
+	@requiresMinion
+	async run(msg: KlasaMessage, [quantity, mixName = '']: [null | number | string, string]) {
+		if (msg.flagArgs.items) {
+			return msg.channel.sendFile(
+				Buffer.from(
+					Herblore.Mixables.map(
+						item =>
+							`${item.name} - lvl ${item.level} : ${Object.entries(item.inputItems)
+								.map(entry => `${entry[1]} ${itemNameFromID(parseInt(entry[0]))}`)
+								.join(', ')}`
+					).join('\n')
+				),
+				`Available Herblore potions and items.txt`
+			);
+		}
+
+		if (typeof quantity === 'string') {
+			mixName = quantity;
+			quantity = null;
+		}
+		const mixableItem = Herblore.Mixables.find(item =>
+			item.aliases.some(alias => stringMatches(alias, mixName))
+		);
+		// const mixableItem = Herblore.Mixables.find(item => stringMatches(item.name, mixName));
+
+		if (!mixableItem) {
+			throw `That is not a valid mixable item, to see the items available do \`${msg.cmdPrefix}mix --items\``;
+		}
+
+		if (msg.author.skillLevel(SkillsEnum.Herblore) < mixableItem.level) {
+			throw `${msg.author.minionName} needs ${mixableItem.level} Herblore to mix ${mixableItem.name}.`;
```

Comment:
always use `return msg.send(` instead of throw, if its just a message you want to show to a user, which is pretty much all instances.